e2e_tests/tests/experiment/experiment.py

```diff
@@ -214,26 +222,62 @@ def experiment_has_completed_workload(experiment_id: int) -> bool:
     if not any(trials):
         return False
 
-    return any(any(s["state"] == "COMPLETED" for s in t["steps"]) for t in trials)
+    for t in trials:
+        for s in t["workloads"]:
+            if ("training" in s and s["training"]["state"] == "STATE_COMPLETED") or (
+                "validation" in s and s["validation"]["state"] == "STATE_COMPLETED"
+            ):
+                return True
+    return False
+
+
+def experiment_config_json(experiment_id: int) -> Dict[str, Any]:
+    murl = conf.make_master_url()
+    certs.cli_cert = certs.default_load(murl)
+    authentication.cli_auth = authentication.Authentication(murl, try_reauth=True)
+    sess = session.Session(murl, "determined", authentication.cli_auth, certs.cli_cert)
+    r = bindings.get_GetExperiment(sess, experimentId=experiment_id)
+    return r.config
 
 
 def experiment_json(experiment_id: int) -> Dict[str, Any]:
```

Comment:
i think we should replace these (this func and all similar onces around) with something like `experiment() -> v1Experiment` or something. to me this is the point of having bindings